scripts/consumer_apps_builder.py

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Before finalizing this, can you add a comment describing at a high level what this script does?